src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -167,6 +167,15 @@ public boolean equals(Object other) {
                 && editRecipeDescriptor.equals(e.editRecipeDescriptor);
     }
 
+    @Override
+    public String toString() {
+        return "EditCommand{"
+                + "targetName=" + targetName
+                + ", targetIndex=" + targetIndex
+                + ", editRecipeDescriptor=" + editRecipeDescriptor
+                + '}';
+    }
+
```

Comment:
Just generally asking if we ever need a toString for any of the commands

Reply:
Only use case is to make testing easier - can print out the values in the EditCommand to check which values didn't get parsed properly etc

Can remove once tests are finalised!

Reply:
@aidanaden oh then i think it's fine ah I think